### ISSUE TITLE
Global submission error

### ISSUE THREAD

**Issue by ToddVolkening** [agrc/electrofishing#218]
"Error unable to submit" message needs to point out where the error occurred.  We were unable to track down the cause of the error.  We had to refresh the browser in order to proceed thus causing loss of all entered data.   

~CW and TV

**Comment by ToddVolkening** [comment]
Encountered issue again on 02/05/2020.

**Comment by stdavis** [comment]
This appears to be an issue when multiple people attempt to submit reports at the same time. Something related to closing edit sessions. Here's the error from the ArcGIS Server logs:
```
Error executing tool. NewCollectionEvent Job ID: je130b4e6c91b48b9aae7dba32f658113 : Traceback (most recent call last): File "C:\arcgisserver\directories\arcgissystem\arcgisinput\Electrofishing\Toolbox.GPServer\extracted\v101\scripts\NewCollectionEvent.py", line 94, in edit.stopEditing(True) RuntimeError: start edit session Failed to execute (NewCollectionEvent). Failed to execute (NewCollectionEvent).
```

**Comment by stdavis** [comment]
I've submitted a ticket to Esri support for this. Waiting to hear back.

**Comment by stdavis** [comment]
This was fixed in: fcee76fd2477f9f050bc1f3c7b83d222c0d98662
